doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -2444,6 +2444,12 @@ The issues in this section are related to the :ref:`nrf53_audio_app` application
 
 .. rst-class:: v3-1-0
 
+OCT-3265: Test with nRF5340 DK as sink will cause the I2S to interfere with other signals on the same pins
+  Using the nRF5340 DK to drive an external digital-to-analog converter via I2S will not produce audio.
```

Comment:
As per https://nordicsemi.atlassian.net/wiki/spaces/TECHDOC/pages/120306494/Style+and+usage+guide, we do not use Latin in the documentation. 
```suggestion
  Using the nRF5340 DK to drive an external digital-to-analog converter through I2S will not produce audio.
```